RNTester/js/OutOfTreeTestPlatform/Libraries/Components/StatusBar/StatusBarIOS.dummy.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ * @flow
+ */
+
+'use strict';
+
+module.exports = {};
```

Comment:
why empty here and not `require('UnimplementedView');` as elsewhere?

Reply:
StatusBarIOS isn't a React component

https://github.com/facebook/react-native/blob/master/Libraries/Components/StatusBar/StatusBarIOS.ios.js

None if this code will be ran at all at any point - it is purely for the bundler's sake, so the actual code isn't *that* important. A lot of the shims, like `DummySegmentedControlIOS` are just copied over from Android shims in the RN codebase. Sorry for not making this clearer! I'll edit the main post